File: 32- x এর গুণিতক.c
#include<stdio.h>
int main()
{

    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,n;
        scanf("%d%d",&x,&n);
        if(x>n)
        {
            printf("Invalid!\n");
        }
        else
        {
            for(int i=x;i<=n;i+=x)
            {
                printf("%d\n",i);
            }
        }
        printf("\n");


    }
    return 0;

}

File: 32- x এর গুণিতক.py
t = int(input())

while t!=0:
    a, b = input().split()
    a = int(a)
    b = int(b)
    if a>b:
        print("Invalid!")
    else:
        for i in range(a,b+1,a):
            print(i)
    

    print()
    t-=1
